ext/exiv2/exiv2.cpp

```diff
@@ -106,7 +106,7 @@ extern "C" void Init_exiv2() {
 // Exiv2::Image Methods
 
 static void image_free(Exiv2::Image* image) {
-  delete image;
+  xfree(image);
```

Comment:
This doesn't invoke `Exiv2::Image`'s destructor, which, while currently empty may not always be.